tests/test_orm.py

```diff
@@ -250,9 +250,7 @@ def test_schema_including_dot(engine_testaccount, db_parameters, run_v20_sqlalch
     """
     Base = declarative_base()
 
-    namespace = '{db}."{schema}.{schema}".{db}'.format(
-        db=db_parameters["database"].lower(), schema=db_parameters["schema"].lower()
-    )
+    namespace = f'{db_parameters["database"].lower()}."{db_parameters["schema"].lower()}.{db_parameters["schema"].lower()}".{db_parameters["database"].lower()}'
```

Comment:
So in these cases I'd rather have `format` calls (that is when we repeat arguments).
Now this is in the tests, so it doesn't matter too much, but also performance isn't that big of a priority in (unlike in the actual library).